assignments/arrays.js

```diff
@@ -75,25 +75,61 @@ for (let i = 0; i < inventory.length; i++){
 let lastCar = 0;
 console.log();
 
+for (let i = 0; i < inventory.length; i++){
+    if (inventory[i] === inventory[inventory.length -1]){
```

Comment:
You don't have to loop through each element in `inventory` to get the lastCar. You can just simply access it with `inventory[inventory.length -1]` and assign it to `lastCar` variable. 👌